Lombiq.Tests.UI/Models/UITestManifest.cs

```diff
@@ -38,6 +37,5 @@ private static T GetValueOfType<T>(object instance)
         instance
             .GetType()
             .GetFields(BindingFlags.NonPublic | BindingFlags.Instance)
-            .FirstOrDefault(field => field.FieldType == typeof(T))
-            ?.GetValue(instance) as T;
+            .Find(field => field.FieldType == typeof(T))?.GetValue(instance) as T;
```

Comment:
I missed this formatting change before, please fix:
```suggestion
            .Find(field => field.FieldType == typeof(T))?
            .GetValue(instance) as T;
```